source/jupyter-book-tutorials.rst

```diff
@@ -0,0 +1,8 @@
+Jupyter Book Tutorials
+============================
+
+These tutorials are powered by `Jupyter Book <https://jupyterbook.org/intro.html>`_
+with support from the `Sloan Foundation <https://sloan.org/grant-detail/9231>`_.
+
+- `q2book <https://gregcaporaso.github.io/q2book/front-matter/preface.html>`_
```

Comment:
`: A introduction to QIIME 2 and concepts in microbiome bioinformatics (a work in progress)`